extensions/google-lens/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "google-lens",
+  "title": "Google Lens",
+  "description": "Search the image using Google Lens",
+  "icon": "extension-icon.png",
+  "author": "ridemountainpig",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-with-screenshot",
+      "title": "Search with Screenshot",
```

Comment:
"Take Screenshot and Search" might be a little more user friendly, what do you think?